backend/app/services/inquiries.py

```diff
@@ -36,3 +42,28 @@ def get_inquiries(
 def count_inquiries(*, session: Session) -> int:
     statement = select(func.count()).select_from(Inquiry)
     return session.exec(statement).one()
+
+
+router = APIRouter()
+
+
+@router.put("/api/inquiries/{inquiry_id}", response_model=InquiryPublic)
+def edit_inquiry(
+    inquiry_id: UUID,
+    inquiry_update: InquiryCreate,
+    current_user: dict = Depends(get_current_user),
+    session: Session = Depends(get_session),
+):
+    if not current_user["is_admin"]:
```

Comment:
we don't really have any idea of a user yet.  we have not been designing for users.  you should not be introducing them.  it is not necessary for this ticket.  furthermore, this is not how you protect routes.  route protection is done in the put decorator, not in the function body.

Reply:
removed